internal/repositories/reward/rewards.go

```diff
@@ -8,13 +8,16 @@ import (
 	"github.com/DIMO-Network/identity-api/internal/helpers"
 	"github.com/DIMO-Network/identity-api/internal/repositories/base"
 	"github.com/DIMO-Network/identity-api/models"
+	"github.com/ericlagergren/decimal"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/volatiletech/null/v8"
 	"github.com/volatiletech/sqlboiler/v4/queries/qm"
 	"github.com/volatiletech/sqlboiler/v4/types"
 	"golang.org/x/exp/slices"
 )
 
+var weiVal = decimal.WithContext(decimal.Context{Precision: 20}).SetMantScale(1, -18)
```

Comment:
Maybe a name like `weiPerEther` and a comment about how it's `10^8` (I always get weirded out by the negative).

Reply:
Could you say why the precision matters? I would've naively done `decimal.New(1, -18)`, I think.

Reply:
I don't think it really does since I am rounding the result.